src/test.zig

```diff
@@ -53,6 +53,10 @@ pub const TestContext = struct {
             /// stdout against the expected results
             /// This is a slice containing the expected message.
             Execution: []const u8,
+            /// A header update compiles the input with the equivalent of
+            /// `-Demit_h=true` and tests the produced header against the
```

Comment:
Why `-Demit-h=true`, isn't it `-femit-h`? Or is this a different syntax.

Reply:
No, you're right, that's my bad